Makefile.am

```diff
@@ -65,17 +65,28 @@ INFRA_KEYS = \
 	keys/signing/EIASK1.pub \
 	$(NULL)
 $(INFRA_KEYRING): $(INFRA_KEYS)
-ALL_KEYRINGS += $(INFRA_KEYRING)
-ALL_KEYS += $(INFRA_KEYS)
+INST_KEYRINGS += $(INFRA_KEYRING)
+INST_KEYS += $(INFRA_KEYS)
 
 # Endless public archive keyring for https://sources.endlessm.com/debian/
 PUBARCHIVE_KEYRING = eos-pub-archive-keyring.gpg
 PUBARCHIVE_KEYS = \
 	keys/signing/EPASK1.pub \
 	$(NULL)
 $(PUBARCHIVE_KEYRING): $(PUBARCHIVE_KEYS)
-ALL_KEYRINGS += $(PUBARCHIVE_KEYRING)
-ALL_KEYS += $(PUBARCHIVE_KEYS)
+INST_KEYRINGS += $(PUBARCHIVE_KEYRING)
+INST_KEYS += $(PUBARCHIVE_KEYS)
+
+# GPG keyring just for testing
+NOINST_KEYRING = noinst-keyring.gpg
+NOINST_KEYS = \
+	keys/signing/ESBASK1.pub \
+	$(NULL)
+$(NOINST_KEYRING): $(NOINST_KEYS)
+noinst_DATA = $(NOINST_KEYRING)
```

Comment:
Don't you need to change `keyring_DATA = $(ALL_KEYRINGS)` below to `keyring_DATA = $(INST_KEYRINGS)`?

Reply:
Indeed!